schutzbot/get_civ_config.py

```diff
@@ -165,23 +166,23 @@ def get_modified_methods_str():
     return " or ".join(list(modified_methods))
 
 
-def get_skip_vars():
-    skip_vars = {
+def get_skip_vars() -> Dict[str, str]:
+    skip_vars: Dict[str, str] = {
         "skip_aws": "false",
         "skip_azure": "false",
         "skip_gcp": "false"
     }
 
-    files_changed = get_files_changed()
+    files_changed: List[str] = get_files_changed()
 
     for file in files_changed:
         if "test_suite/generic/.*" in file:
             return skip_vars
 
```

Comment:
@F-X64 @narmaku this PR doesn't change any test_suite/cloud/*.py files, meanwhile, the test_suite are executed, e.g. see [aws-rhel-9.4-ga-x86](https://gitlab.com/redhat/services/products/image-builder/ci/cloud-image-val-ci/-/jobs/9789785811#L2384), meaning skip variable in lines 182-184 is not returned, but it happens much earlier, on likes 180, and it's take as "false" from line 170, is it? Otherwise, why were test triggered? 

Then, on line 179, why do we check if test_suite/generic is changed and then return skip_vars? this logic I don't follow completely.

Reply:
Let's cover this in our pair programming call today.